src/main/java/mekanism/common/integration/multipart/MultipartTileNetworkJoiner.java

```diff
@@ -0,0 +1,83 @@
+package mekanism.common.integration.multipart;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import io.netty.buffer.ByteBuf;
+import mcmultipart.api.container.IMultipartContainer;
+import mcmultipart.api.slot.EnumFaceSlot;
+import mcmultipart.api.slot.IPartSlot;
+import mcmultipart.api.world.IMultipartBlockAccess;
+import mekanism.common.base.ITileNetwork;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.world.IBlockAccess;
+
+public class MultipartTileNetworkJoiner implements ITileNetwork
```

Comment:
Quite a few, but the only places I can see that impacting multiparts are:
- `TileEntitySidedPipe.update()`
- `TileEntityLogisticalTransporter.onConfigure(EntityPlayer, int, EnumFacing)`
- `TileEntityThermodynamicConductor.sendTemp()`